examples/pennylane/4_Simulation_of_noisy_quantum_circuits_on_Amazon_Braket_with_PennyLane/4_Simulation_of_noisy_quantum_circuits_on_Amazon_Braket_with_PennyLane.ipynb

```diff
@@ -27,9 +27,9 @@
    "source": [
```

Comment:
<p>nit: confirming the <em>variational quantum eigensolver</em> hyperlink broken as we haven't merged the changes yet in main? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Correct - this VQE location is updated as part of this PR